src/runtime_src/core/pcie/tools/cloud-daemon/mpd.service.in

```diff
@@ -6,7 +6,7 @@ ConditionDirectoryNotEmpty=/dev/xfpga
 
 [Service]
 Type=simple
-ExecStart=${XRT_INSTALL_DIR}/bin/mpd
+ExecStart=${CMAKE_INSTALL/PREFIX}/${XRT_INSTALL_BIN_DIR}/mpd
```

Comment:
CMAKE_INSTALL_PREFIX  :-)

Reply:
silly me. I also ran a build and happened just verifying the other one.
thanks for the catch
changed.